src/features/base-layout/components/header/burger/burger.module.scss

```diff
@@ -58,6 +58,7 @@
 
   &.open {
     display: block;
+    background-color: white;
```

Comment:
Do we really need this?

Reply:
I 

> Do we really need this?

I think so, but it's a matter of taste and have to choose

Reply:
Will the component change its color if we remove this? I suggest not to write unnecessary styles, this can lead to overriding styles.